charts/vector/Chart.yaml

```diff
@@ -18,11 +18,11 @@ maintainers:
   - name: Datadog
     email: vector@datadoghq.com
 icon: https://vector.dev/press/vector-icon.svg
-appVersion: "0.42.0-distroless-libc"
+appVersion: "0.43.0-distroless-libc"
```

Comment:
I think you'll need to bump the chart version to 0.38.0 here because 0.37.0 has already been released (CI checks that chart versions have been bumped).

Reply:
done in https://github.com/vectordotdev/helm-charts/pull/430/commits/fc86b0502f723b5993a16faf18ea4e4acf3da92d